swagger.json

```diff
@@ -5138,23 +5138,34 @@
             }
         },
         "DocumentField": {
+            "allOf": [
+                {
+                    "type": "object",
+                    "required": [
+                        "label"
+                    ],
+                    "description": "A field of a document. A field can be a string type, number type, or photo type, which you can identify from its valueType. Between stringValue, numberValue, and photoValue, only one can exist for a single document field depending on the valueType.",
```

Comment:
nit: should we be writing our documentation in second person?

Reply:
@silverspace Not sure what we prefer.

Reply:
Just checked, doesn't seem like it's used elsewhere (except for a broad overview of our API)